src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int NUMBER_OF_FIGURE_TYPES = 5;
+    private static final int SIZE_LIMIT = 99;
+    private static final String COLOR_OF_CIRCLE = Color.WHITE.name();
+    private static final int RADIUS_OF_CIRCLE = 10;
+    private Random random = new Random();
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        int randomNumberToChooseFigure = random.nextInt(NUMBER_OF_FIGURE_TYPES);
+        int firstRandomNumberToSetSizes = random.nextInt(SIZE_LIMIT) + 1;
```

Comment:
Let's create separate method and use it during initialisation each type of figure
```
private int randomSize() {
        return random.nextInt() * SIZE_LIMIT;
    }
```